board/templates/home.html

```diff
@@ -1,472 +1,1016 @@
+                  </li>
+                </ul>
+              </div>
+              <div class="menu_section">
+                <h3>Deployments</h3>
+                <ul class="nav side-menu">
+                  <li><a><i class="fa fa-check"></i> Deployed <span class="fa fa-chevron-down"></span></a>
+                    <ul class="nav child_menu">
+                      <li><a href="form.html">Deploy_1</a></li>
+                      <li><a href="form.html">Deploy_2</a></li>
+                      <li><a href="form.html">Deploy_3</a></li>
+                      <li><a href="form.html">Deploy_4</a></li>
+                    </ul>
+                  </li>
+                  <li><a><i class="fa fa-gear"></i> Configurations <span class="label label-success pull-right">Coming Soon</span> </a>
+                  </li>
+                  <li><a><i class="fa fa-search"></i> Monitoring  <span class="label label-success pull-right">Coming Soon</span></a>
+                  </li>
+                </ul>
+              </div>
+            </div>
+            <!-- /sidebar menu -->
+
+            <!-- /menu footer buttons -->
+            <div class="sidebar-footer hidden-small">
+              <a data-toggle="tooltip" data-placement="top" title="Logout" href="{% url 'logout' %}">
+                <span class="glyphicon glyphicon-off" aria-hidden="true"></span>
               </a>
-            </li>
-            <li class="nav-item">
-              <a class="nav-link" href="#">About</a>
-            </li>
-            <li class="nav-item">
-              <a class="nav-link" href="https://github.com/mohit-chawla/recboard">Github</a>
-            </li>
-            {% if user.is_authenticated %}
-            <li class="nav-item">
-              <a class="nav-link active" href="#"><i class="fa fa-user-circle"></i> {{ user.username }}</a>
-            </li> 
-            
-            <li class="nav-item">
-              <a class="nav-link" href="{% url 'logout' %}"> Log out</a>
-            </li>
-            {% endif %}
-          </ul>
+            </div>
+            <!-- /menu footer buttons -->
+          </div>
         </div>
-      </div>
-    </nav>
-    <body>
-        
-        <div class="container-fluid">
-            <!-- IMPORTANT enctype attribute! -->
-            <div class="row top-buffer">
-                <div class="col-md-12">
-                </div>
+
+        <!-- top navigation -->
+        <div class="top_nav">
+          <div class="nav_menu">
+            <nav>
+              <div class="nav toggle">
+                <a id="menu_toggle"><i class="fa fa-bars"></i></a>
+              </div>
+
+              <ul class="nav navbar-nav navbar-right">
+                <li class="">
+                  <a href="javascript:;" class="user-profile dropdown-toggle" data-toggle="dropdown" aria-expanded="false">
+                    <img src="{{STATIC_URL}}/static/img/mohit.jpg" alt="">Mohit Chawla
```

Comment:
Same comment about displaying this using `{{ user.username }}`

Reply:
same reply